### ISSUE TITLE
Support remote lookups of webfinger accounts and autolink them in outbound Fediverse posts

### ISSUE THREAD

**Issue by decentral1se** [owncast/owncast#3659]
### Share your bug report, feature request, or comment.

I can't seem to link to it, as it gives me this [link](https://cast.coopcloud.tech/federation/qUdCRFbIg) which now just links to the offline stream page but a screengrab of it is:

![image](https://github.com/owncast/owncast/assets/1991377/7b4df93b-6e29-4f99-a4f3-2c8c0b196f12)

where `@decentralse...` & `@3wordchant` should be clickable? This was reported on different clients: semaphore web UI, mastodon web UI.

**Comment by gabek** [comment]
That's true, it doesn't currently support looking up remote accounts in posts.
